cleansweep/volunteers/views.py

```diff
@@ -14,7 +15,18 @@ def init_app(app):
 
 @plugin.place_view("/volunteers", permission="view-volunteers")
 def volunteers(place):
-    return render_template("volunteers.html", place=place)
+    try:
+        page_arg_value = int(request.args.get('page', 1))
+        page = page_arg_value if page_arg_value > 0 else 1  # You never know...
+    except ValueError:
+        page = 1
+    total_count = place.get_member_count()
+    limit = 50
+
+    pagination = Pagination(total=total_count, page=page, per_page=limit,
```

Comment:
Using `Pagination` object seems to be a standard practice in Django and Flask. I'm okay with that, but I would prefer the try-except to go to some utils module. Merging it for now.